spec/support/requests/requests_helper.rb

```diff
@@ -4,4 +4,12 @@ def json
       @json ||= JSON.parse(response.body)
     end
   end
+
+  module LoginHelpers
+    def token_auth
+      {
+        'HTTP_AUTHORIZATION' => ActionController::HttpAuthentication::Token.encode_credentials(ENV['API_SECRET'])
```

Comment:
Line is too long. [113/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.